### ISSUE TITLE
opening configuration page

### ISSUE THREAD

**Issue by maxwell8888** [shanalikhan/code-settings-sync#1024]
Hi

I have settings sync working correctly on one laptop, but I am struggling to get it synced with another. I tried joining the slack but it said the link is no longer valid.
In the instructions it says that: "Settings Sync Configuration page will be opened automatically on code start and requires two things to setup"
However this is not happening for me. How can I open it manually?

Thanks

**Comment by shanalikhan** [comment]
>  However this is not happening for me. How can I open it manually?

If it doesnt open automatically, it will always open on upload/download if Settings Sync is not configured.

I suggest you to Reset the Extension Settings by the command `Sync: Reset Extension Settings` and then reload the code to see if the page opens for you to configure.

There is some issue if it doesnt opens for you. You need to post the image of developer console log on selecting upload or download command so i can see the error behind.

**Comment by bernardoadc** [comment]
I had the same issue, the welcome page did not showed up. Reading the docs I was able to find syncLocalSettings file, remove token there, and with new update command the page has opened. Creating manually a personal token asked a lot of details, so this path is not so nice.

Perhaps a command just to show the welcome page? It wouldn't be so hard and pretty straightforward. I didn't go for the reset option 'cause was afraid I would lose it all (perhaps some clarification in readme would suffice too)

**Comment by shanalikhan** [comment]
> Reading the docs I was able to find syncLocalSettings file, remove token there, and with new update command the page has opened.

Welcome page appears when you dont have token setup or you are setting as new user.

> I didn't go for the reset option 'cause was afraid I would lose it all (perhaps some clarification in readme would suffice too)

What do you suggest in terms of improvement in Readme. You can send PR or suggest me here, i will be more happy to do that. Always looking to improve the documentation.

> Perhaps a command just to show the welcome page? 

What would be the reason to show the welcome page by command. One option can be to replace `Sync : How to Configure` command with `Sync : Open Welcome Page`

**Comment by bernardoadc** [comment]
> Welcome page appears when you dont have token setup or you are setting as new user.

Yes, but my token was revoked somehow, and I couldn't setup again in welcome page. (this leads to reset clarification below)

> What do you suggest in terms of improvement in Readme

Currently there's only 'Select "> Sync : Reset Extension Settings" in the Command Palette to reset your settings', but it does not tells me what this does. Will it delete syncLocalSettings.json? Clear gist id? Clear github token? Will it ask me again those things? Or will I have to go elsewhere later to redo this. 

> One option can be to replace Sync : How to Configure command with Sync : Open Welcome Page

Well, "how to configure" will show me a readme, while "open welcome page" should have login button. Or perhaps this is already the "> Sync : Advanced Options > Open Settings Page" option. This could go "up a level", i.e., "Sync > Open Settings Page", and there a button for "renew token".


These are only thoughts and insights for you to reflect upon, from a user perspective, by no means it diminishes this great extension :)

**Comment by kremens** [comment]
"If it doesn't open automatically, it will always open on upload/download if Settings Sync is not configured."

This doesn't appear to be true. I'm having the same issues as the users above. This issue shouldn't be closed, as the documentation causes confusion by saying: "Settings Sync Configuration page will be opened automatically on code start," because, again, for some users this doesn't happen.

However, the way out of the confusion is: Sync : Advanced Options > Open Settings Page.
